Makefile

```diff
@@ -37,74 +37,61 @@ mgmt-destroy:
 	&& . ./devops/scripts/load_terraform_env.sh ./devops/.env \
 	&& cd ./devops/terraform && ./destroy.sh
 
+# A recipe for building images. Parameters:
+# 1. Image name suffix
+# 2. Version file path
+# 3. Docker file path
+# 4. Docoer context path
+# Example: $(call build_image,"api","./api_app/_version.py","api_app/Dockerfile","./api_app/")
+define build_image
```

Comment:
Don't get me wrong, I ❤️ this, the repeating code in the makefile were making me shudder. However, I wonder whether this would be better in a script with some proper checking of the existence of the variables. As when it goes wrong, debugging a makefile is not fun! Happy to leave it too

Reply:
Not sure what will be in the script and how will that play nicely with the whole way env is loaded and used.
Anyway, I didn't much of the logic here, just centralized it with a few params.
Feels to me that revamping make might be a separate thing but willing to discuss...

Reply:
Agreed - I think this makefile will get changed incrementally anyway. It's a step in the right direction